DB/api/db.py

```diff
@@ -184,6 +243,23 @@ def perform_query(query, params):
 	finally:
 		if con:
 			con.close()
+def format_energy_list(rows):
+	"""
+	Formats result set for energy_list query
+
+	:param results:
+	:return:
+	"""
```

Comment:
is this a good compromise for now? I don't think my group members doing the graphing will accept data in any other format at this point. We have a demo on Tuesday that needs to work.